apps/namadillo/src/polyfills.ts

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore: Unreachable code error
+BigInt.prototype.toJSON = function (): string {
```

Comment:
Just a question, but where is this getting used?

Reply:
We've defined a few bigint types in the @anomaorg/namada-indexer-client api. I haven't checked how the react-query handles changes in their dependencies, but probably they're stringified and hashed, and that's why this polyfill is needed